### ISSUE TITLE
Allow titles with math notation

### ISSUE THREAD

**Issue by ShahriarKh** [leonhma/obsidian-functionplot#62]
### What's your feature?

Since we are dealing with math, I believe it's really required to be able to have math equations in the title.
For example:
`title: $\lim_{x\to a}$`

**Comment by leonhma** [comment]
Agree. We could use [renderMath](https://marcus.se.net/obsidian-plugin-docs/reference/typescript/functions/renderMath) to render the equation into the title element. Maybe just skip this step if no mathjax is detected. Great idea btw.

**Comment by ShahriarKh** [comment]
So, this:
https://github.com/leonhma/obsidian-functionplot/blob/2a69384215690967a02c7b39ef644b11a3aadd5a/src/common/utils.ts#L19
should change to something like 
`title: ${options.title is math ? renderMath(options.title) : options.title}`

**Comment by leonhma** [comment]
Yes. Either that or we could support the `$ ... $` syntax obsidian uses. I don't exactly know how renderMath works but we should have a function that renders text to html element and also handles math. Then we could use it as a component for the title and legends.
